lib/__tests__/test-stack.ts

```diff
@@ -30,9 +71,12 @@ export class TestStack extends Stack {
     // BUILD
     //
 
+    const superchain = LinuxBuildImage.fromDockerRegistry('public.ecr.aws/c4i6u9i2/jsii/superchain:1-buster-slim-node14');
```

Comment:
Unrelated, but changed this to avoid DockerHub throttling. Though I just now realized the image hasn't been updated in about a month...

@RomainMuller Have we stopped publishing to ECR?

See https://gallery.ecr.aws/c4i6u9i2/jsii/superchain